docs/src/main/tut/docs/alternative_effects.md

```diff
@@ -11,62 +11,105 @@ title: "Side effects"
 
 ```scala
 import neotypes.implicits.mappers.results._ // Brings the implicit ResultMapper[String] instance into the scope.
-import neotypes.implicits.syntax.session._ // Provides the asScala[F[_]] extension method.
 import neotypes.implicits.syntax.string._ // Provides the query[T] extension method.
 import scala.concurrent.Future
+import scala.concurrent.ExecutionContext.Implicits.global
 
-val s = driver.session().asScala[Future]
+val program: Future[String] = for {
+  driver <- GraphDatabase.driver[Future]("bolt://localhost:7687", AuthTokens.basic("neo4j", "****"))
+  session <- driver.session()
+  data <- "match (p:Person {name: 'Charlize Theron'}) return p.name".query[String].single(session)
```

Comment:
Not that I care too much about `Future` :smile: but if this is the official example shouldn't it be shown that the session and driver are properly closed? In this example, if there's an error in the query then there's a session / driver leak.

Reply:
Uhm... you are right, I really haven't used `Future` in my life, I am just used to the safety of `IO` 😅 

Would this be a better example?

```scala
val program = for {
  driver <- GraphDatabase.driver[Future]("bolt://localhost:7687", AuthTokens.basic("neo4j", "****"))
  result <- driver.readSession { session =>
    "match (p:Person {name: 'Charlize Theron'}) return p.name".query[String].single(session)
  }.recoverWith {
    case tryResult => driver.close().recoverWith {
      case _ => Future.fromTry(tryResult) 
    }
  }
} yield result
```

Reply:
Not sure what's idiomatic with `Future` tbh...